Remove unnecessary docker scripts: (#2)
aiidateam/aiida-docker-base: README.md

# Docker Hub repository

> The docker image is built automatically on Docker Hub once new changes were pushed to the `master` or `develop` branches of the corresponding GitHub repository.

Reviewer: ```suggestion
The docker image is built automatically on Docker Hub once new changes are pushed to the `master` or `develop` branches of this repository.
```

> The image pushed to the `master` will be tagged as `latest`, the image pushed to the `develop` branch will be tagged as `develop`.

Reviewer: ```suggestion
The `master` branch is available under the docker tag `latest`, while the `develop` branch is available under the docker tag `develop`.
```

> In addition to that any new version tag that is pushed to the repository will trigger Docker Hub to build the image with the same tag as on the GitHub.

Reviewer: ```suggestion
In addition, any git tag pushed to the repository will trigger a build on Docker Hub the same docker tag.
```
